conans/client/tools/files.py

```diff
@@ -70,6 +70,13 @@ def unzip(filename, destination=".", keep_permissions=False, pattern=None, outpu
             filename.endswith(".tbz2") or filename.endswith(".tar.bz2") or
             filename.endswith(".tar")):
         return untargz(filename, destination, pattern)
+    if filename.endswith(".gz"):
+        import gzip
+        with gzip.open(filename, 'rb') as f:
+            file_content = f.read()
+        target_name = filename[:-3] if destination == "." else destination
```

Comment:
Here the destination is expected to be a file (not a directory like in the other cases), the docstring for this function doesn't match. If given a directory this function will fail in the `load` with a `IsADirectoryError`. I'm not sure if we want to handle it better.

Reply:
Updated docstring.

I have tried passing a folder as argument, and it failed (py2, Win), with ``IOError: [Errno 13] Permission denied: 'mytemp'``. I think in either case the error should be clear enough, catching and re-raising a ConanException is a bit overkill for this case.

Reply:
No, don't need to reraise, it was just about the docstring (it is a documented tool). Don't forget to update it: https://docs.conan.io/en/latest/reference/tools.html#tools-unzip 😉